### PR TITLE
Added a tutorial page(link) to guide

### PR DESCRIPTION
**Pull request by lordzden**
I added a link tutorial to help guide those first time testers. This tutorial can help them save time. The reason I posted this because I happened to me the first time and I have to pull out several web documents just to understand [how the validation works](https://asknoypi.com/accelerated-mobile-pages-validator/). 

THanks

### CODE REVIEW

**Comment by lordzden** [comment]
I signed it!

**Comment by lordzden** [comment]
Hello,
This is my first time contributing. Please help me push my edit.

Thanks.

- Baloydi

File: README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ fast.
 
 Learn more about [validating your AMP pages](https://www.ampproject.org/docs/guides/validate.html).
 
+Tutorial on [how to validate AMP pages](https://asknoypi.com/accelerated-mobile-pages-validator/).
```
Comment (lordzden) [inline README.md]: I added a tutorial link so first timers can understand how the validation works. If you are not really good in searching you will end up confused on how the validation takes place on Chrome. The tutorial contains images that may also help testers navigate the testing console on Chrome.

**Comment by lordzden** [comment]
Hello,
What should I do next on this edit?

Thanks

**Comment by rudygalfi** [comment]
@lordzden Thanks for the PR, and thanks for creating a resource to help others with addressing validation issues.

Rather than linking off to another page, we'd like suggestions on how we can improve existing pages on topics. https://www.ampproject.org/docs/guides/validate.html seems to already overlap heavily with the content at the link you proposed adding. Do you think you could update this PR to make updates to existing guide pages?

**Comment by lordzden** [comment]
I understand @rudygalfi thanks for adding your response.

**Pull request closed without merging**